.github/workflows/releasability.yml

```diff
@@ -3,6 +3,8 @@ name: Releasability status
   check_suite:
     types:
       - completed
+  workflow_dispatch: 
```

Comment:
I am not sure does this workflow_dispatch does.

Reply:
It allows running the action manually in the menu, like when re-deploying Peach.